README.adoc

```diff
@@ -52,9 +52,14 @@ Using `-b` (or `--backend`) the following formats are supported:
 
 == Installation
 
-This is a standard Go package, and it installs like you might expect.
+To build libasciidoc and make it available on the command line, do this:
 
-To install it in your `$GOPATH` (assuming `$GOPATH/bin` is on your path), just use `go install`:
+    $ git checkout github.com/bytesparadise/libasciidoc
+    $ cd libasciidoc
+    $ make install
+    $ sudo ln -s  ./bin/libasciidoc /usr/bin/libasciidoc
```

Comment:
the `make install` goal already copies the binary into `$GOPATH/bin` so I think that we should ask the users to ensure that `$GOPATH/bin` in `$PATH`, instead of making a symlink into a directory of `$PATH`. Also, `sudo ln -s ./bin/libasciidoc /usr/bin/libasciidoc` won't work for Windows users :(